Wizardry/Core/UnitStatusGetter/source/PowGetter.c

```diff
@@ -86,6 +86,17 @@ int PowGetterSkills(int status, struct Unit * unit)
             status += 5;
 #endif
     }
+    
+#if (defined(SID_Resolve) && (SID_Resolve < MAX_SKILL_NUM)) 
+        if (SkillTester(unit, SID_Resolve))
+        {
+            if (((GetUnitMaxHp(unit) % 2 == 0) && (GetUnitCurrentHp(unit) < GetUnitMaxHp(unit)/2)) ||
+                ((GetUnitMaxHp(unit) % 2 != 0) && (GetUnitCurrentHp(unit) -1 < GetUnitMaxHp(unit)/2)))
+                {
+                    status += status/2;
```

Comment:
add space for "/"， Please be consistent with the code style of other parts.

Reply:
Done